phone/src/apps/bank/hooks/useBankService.ts

```diff
@@ -21,8 +22,8 @@ export const useBankService = () => {
     });
   };
 
-  useNuiEvent('BANK', 'setTransaction', setTransaction);
-  useNuiEvent('BANK', 'setCredentials', setCredentials);
+  useNuiEvent('BANK', BankEvents.SEND_TRANSFERS, setTransaction);
+  useNuiEvent('BANK', BankEvents.SEND_CREDENTIALS, setCredentials);
   useNuiEvent('BANK', 'setAlert', handleAddAlert);
```

Comment:
please replace all the events

Reply:
You....you just said...alright.

Reply:
These methods won't work in the useNuiEvent, as the appName factory fails to parse them correctly if they have a prefix of `npwd`

Reply:
they already had a colon so it should be fine tho

Reply:
already fixed messages like this @TasoOneAsia